Update app-insights-sampling.md

zakimaksyutov opened the pull request:
Users could not find that sampling doesn't affect metrics (based on ICM ticket)

azurecla commented:
Hi __@arunvinoth__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

zakimaksyutov committed:
Update app-insights-sampling.md

Users could not find that sampling doesn't affect metrics (based on ICM ticket)

GitHubber17 committed:
Update app-insights-sampling.md
